### PR TITLE
Adjust the Cargos input in Usuarios

### PR DESCRIPTION
**Pull request by LinconOssa**
The input related to “Cargos” in “Usuarios” should be a SearchUserCard

### CODE REVIEW

**Review by ANGIECATHERINEP — APPROVED** [review]
The request was made correctly

File: src/pages/privileges/outlets/users/complete-invitation/index.tsx [hunk 1]
```diff
@@ -151,6 +154,22 @@ function CompleteInvitation() {
     },
   };
 
+  const [positionsOptions, setPositionsOptions] = useState<
```
Comment (carin91) [inline src/pages/privileges/outlets/users/complete-invitation/index.tsx:157]: The states must be created at the beginning of the component.

File: src/pages/privileges/outlets/users/complete-invitation/index.tsx [hunk 2]
```diff
@@ -151,6 +154,22 @@ function CompleteInvitation() {
...
+    Record<string, unknown>[]
+  >([]);
+
+  useEffect(() => {
```
Comment (carin91) [inline src/pages/privileges/outlets/users/complete-invitation/index.tsx:161]: the react hooks must be created at the begining of the component. Usually, the useEffect hook appear after the states.

**Review by carin91 — APPROVED** [review]

File: src/components/feedback/InteractiveModal/types.ts [hunk 3]
```diff
@@ -20,10 +20,7 @@ interface InteractiveModalProps {
   labels?: Field[];
   infoTitle?: string;
   actionsTitle?: string;
-  searchData?:
-    | { [key: string]: string }
-    | Record<string, string | number>
-    | any;
+  searchData: { [key: string]: string } | Record<string, string | number> | any;
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/components/feedback/InteractiveModal/types.ts]: Resolved

File: src/pages/privileges/outlets/users/complete-invitation/index.tsx [hunk 4]
```diff
@@ -6,10 +6,12 @@ import { payrollsFormInvitation } from "@mocks/apps/privileges/invitations/payro
 import { projectsFormInvitation } from "@mocks/apps/privileges/invitations/projectsForm.mock";
 import { IVerificationData } from "@pages/privileges/outlets/users/complete-invitation/interface";
 import { EMessageType } from "@src/types/messages.types";
-import { useState } from "react";
+import { useEffect, useState } from "react";
```
[the remark this answers is not in the document]
Comment (LinconOssa) [reply src/pages/privileges/outlets/users/complete-invitation/index.tsx]: Resolved

File: src/pages/privileges/outlets/users/complete-invitation/index.tsx (hunk 1 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/users/complete-invitation/index.tsx]: Resolved

File: src/pages/privileges/outlets/users/complete-invitation/index.tsx (hunk 2 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/users/complete-invitation/index.tsx]: Resolved

**Pull request merged**